scripts/ensure-env.js

```diff
@@ -0,0 +1,5 @@
+var fs = require("fs");
```

Comment:
based on https://github.com/mozilla/id.webmaker.org/pull/428#issuecomment-279095560 we'll need a check here to make sure we're on a local dev's machine, not on heroku...